apps/condo/domains/news/components/NewsForm/CreateNewsForm.tsx

```diff
@@ -189,15 +188,17 @@ export const CreateNewsForm: React.FC = () => {
         },
     })
 
-    const templates = isNewsItemTemplatesFetching ? null : newsItemTemplates
+    const templates = isNewsItemTemplatesFetching || !newsItemTemplates.length ? null : newsItemTemplates
```

Comment:
if newsItemTemplates is not an array – this code will throw unhandled exception and page will fail

Do we need to check newsItemTemplates ?